tox.ini

```diff
@@ -11,7 +11,7 @@ requires =
 [testenv]
 deps = -e .[test]
 commands =
-    pytest tests --ignore tests/test_nn.py -s {posargs}
+    pytest tests --ignore tests/test_nn.py {posargs}
```

Comment:
is this to speed things up? or, what's the reason for this change?

Reply:
`-s` is the pytest argument for not capturing stdout. It's not really needed so I removed it.